api-client/src/models/__tests__/LocationModel.spec.ts

```diff
@@ -1,126 +1,168 @@
-import { PoiFeature } from '../../maps'
+import {PoiFeature} from '../../maps'
 import LocationModel from '../LocationModel'
 
 describe('LocationModel', () => {
-  describe('location', () => {
+    const thumbnail = 'thumbnail'
+    const path = '/augsburg/de/locations/erster_poi'
+    const urlSlug = 'erster_poi'
+    const expectedGeoJsonMarkerFeature: PoiFeature = {
+        type: 'Feature',
+        geometry: {
+            type: 'Point',
+            coordinates: [31.133859, 29.979848]
+        },
+        properties: {
+            id: 1,
+            title: 'Test',
+            symbol: 'marker_15',
+            thumbnail: 'thumbnail',
+            path: '/augsburg/de/locations/erster_poi',
+            urlSlug,
+            address: 'Wertachstr. 29'
+        }
+    }
     it('should return name if town, address and name is null', () => {
-      expect(
-        new LocationModel({
-          id: 1,
-          name: 'test',
-          country: null,
-          region: null,
-          state: null,
-          address: null,
-          town: null,
-          postcode: null
-        }).location
-      ).toBe('test')
+        expect(
```

Comment:
Please revert, wrong indentation

Reply:
The commit which changed the indention was reverted. I took a look again and ran prettier:fix. If the indention ist still wrong maybe you should try to fix it?